packages/browser/src/backend.ts

```diff
@@ -77,6 +77,11 @@ export class BrowserBackend extends BaseBackend<BrowserOptions> {
       this._newTransport = makeNewFetchTransport({ requestOptions, url });
       return new FetchTransport(transportOptions);
     }
+
+    this._newTransport = makeNewXHRTransport({
+      url,
```

Comment:
Think it’s a bad rebase, we’ll need to fix this!

Reply:
sorry, should be resolved now